lib/pymedphys/docs/libusers/index.rst

```diff
@@ -0,0 +1,39 @@
+======================
+Overview
+======================
+
+Resources to assist users of the PyMedPhys library. If you're experienced with
+Python but brand new to PyMedPhys, `start here`_.
+|
+The **Library Users Guide** is separated into following sub-categories:
+
+- *Tutorials*
```

Comment:
Is it worth having hyperlinks here?

![image](https://user-images.githubusercontent.com/6559099/105945624-be214600-60b9-11eb-9779-71c907f7b19c.png)

Reply:
Also, was the following intentional:

![image](https://user-images.githubusercontent.com/6559099/105946108-af875e80-60ba-11eb-9cbe-1b3b11e7b249.png)

I personally much prefer to not put these spacing items in anyway. If they should be there, maybe they should be done with CSS so that it's globally applied...

Reply:
Yep, the hyperlinks are a good point! Will add :)

I do think that the current theme has insufficient whitespace before headings & subheadings. Very happy to do the CSS - but do you know off the top of your head how? It took me a while even to work out the resizing of the headings...

Reply:
Playing with the numbers after margin in the following should do the trick I think:

```css
h1 { font-size: "something here"; margin: .5em 0; }
```